### DISCUSSION TITLE
Translation SaaS softwares

### DISCUSSION THREAD

**Discussion by slorber** [facebook/docusaurus#4065] [General]
# Translation SaaS softwares

We have official doc for Crowdin (and dedicated issue: https://github.com/facebook/docusaurus/discussions/4052)

Community-driven issue to discuss translation SaaS softwares and how to integrate them with Docusaurus i18n system.

The Docusaurus core team can't team all existing translation softwares, as there are too many.

If you test another translation SaaS software and it is successful for your usecase, please tell us how this works, and we might add some official documentation for it if it's a great fit for Docusaurus.

**Comment by JEETJM** [comment]
If you want to manage translations for a Docusaurus documentation site, you can use several SaaS tools.

Some commonly used options are:

• Crowdin  
• Lokalise  
• Phrase  
• Transifex  

These tools integrate well with GitHub and allow contributors to translate documentation easily.

Docusaurus also has built-in i18n support which you can enable in the configuration.

Documentation:
https://docusaurus.io/docs/i18n/introduction
